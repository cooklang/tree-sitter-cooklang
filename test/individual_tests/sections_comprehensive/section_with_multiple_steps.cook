= Main Course

Heat the #oven{} to 180Â°C.

Place @chicken{1%kg} in #roasting pan{}.

Cook for ~{45%minutes}.