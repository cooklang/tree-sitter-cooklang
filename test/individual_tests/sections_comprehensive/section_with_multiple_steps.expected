(recipe
  (section
    header: (section_header
      name: )
    content: (step
      (plain_text)
      (cookware
        name: (cookware_text))
      (plain_text))
    content: (step
      (plain_text)
      (ingredient
        name: (ingredient_text)
        (amount
          (quantity)
          (units)))
      (plain_text)
      (cookware
        name: (cookware_text)))
    content: (step
      (plain_text)
      (timer
        (amount
          (quantity)
          (units))))))